backend/migrations/0003_create_users_table.down.sql

```diff
@@ -0,0 +1,2 @@
+-- Add down migration script here
+DROP TABLE IF EXISTS users;
```

Comment:
```suggestion
DROP TABLE IF EXISTS user;
```
https://www.sqlshack.com/learn-sql-naming-conventions/